openshift/release/update-to-head.sh

```diff
@@ -40,6 +40,7 @@ git checkout upstream/main -B release-next
 # Update openshift's main and take all needed files from there.
 git fetch openshift main
 git checkout openshift/main $custom_files
+go get github.com/markbates/pkger/cmd/pkger
 pkger
```

Comment:
Switching L43-44 to the script Markus added might be a better alternative, as it's already leveraging vendored pkger.

https://github.com/knative-sandbox/kn-plugin-func/blob/main/hack/update-pkger.sh

E.g.:
```
./hack/update-pkger.sh
```

Reply:
That's just an idea I had when saw the change. (nit picking :))

Reply:
It's probably the right approach. Jefferson's fix should be fine for the short term though.